src/test/abilities/wimp_out.test.ts

```diff
@@ -632,4 +632,35 @@ describe("Abilities - Wimp Out", () => {
     const hasFled = enemyPokemon.switchOutStatus;
     expect(isVisible && !hasFled).toBe(true);
   });
+  it("wimp out will not skip battles when triggered in a double battle", async () => {
+    const wave = 2;
+    game.override
+      .enemyMoveset(Moves.SPLASH)
+      .enemySpecies(Species.WIMPOD)
+      .moveset([ Moves.MATCHA_GOTCHA, Moves.FALSE_SWIPE ])
+      .startingLevel(50)
+      .enemyLevel(1)
+      .battleType("double")
+      .startingWave(wave);
+    await game.classicMode.startBattle([
+      Species.RAICHU,
+      Species.PIKACHU
+    ]);
+    const enemyPokemon = game.scene.getEnemyField();
+    const wimpod0 = enemyPokemon[0];
+    const wimpod1 = enemyPokemon[1];
```

Comment:
```suggestion
    const [ wimpod0, wimpod1 ] = game.scene.getEnemyField();
```
Thankfully TS has a simpler way to write this. Array/object destructuring is pretty handy, you can also do something like `const { arena, currentBattle, ... } = sceneVariable;`. Would have just made a note for the future but I also wanted to change something else so here we are lol.

cf https://www.typescriptlang.org/docs/handbook/variable-declarations.html#destructuring